component-samples/demo/scripts/package/svi-insructions.txt

```diff
@@ -0,0 +1,4 @@
+[ {"filedesc": "device-package.tar.gz", "resource": "device-package.tar.gz"},
+  {"exec": ["tar","-xf","device-package.tar.gz"] },
```

Comment:
can we update it to work with CSDK

[ {"filedesc": "device-package.tar.gz", "resource": "device-package.tar.gz"},
  {"exec": ["/usr/bin/tar","-xf","device-package.tar.gz"] },
  {"exec_cb": ["/bin/bash","onboard-device.sh"] },
  {"fetch": "owner-package.tar.gz"}]